website/source/layouts/api.erb

```diff
@@ -3,6 +3,7 @@
     <ul class="nav docs-sidenav">
       <li<%= sidebar_current("api-overview") %>>
         <a href="/api/index.html">Overview</a>
+        <%= api_toc("index") %>
```

Comment:
Let's remove the TOC on the index. It's nice, but it pushes the main API endpoint docs down page pretty far and they're what people are probably looking for.

Reply:
yeah, I was wondering about that, will do, thanks!